app.py

```diff
@@ -86,8 +86,7 @@ def update_dgraph_metrics():
     @api.background.task
     def update_openshift_metrics():
         _LOGGER.debug("updating OpenShift metrics")
-
-        print("No openshift metrics")
+        get_thoth_graph_sync_jobs("thoth-test-core")
```

Comment:
Can this be configurable (picked from env variables)?

Reply:
I added:
```
if namespace is None:
        namespace = os.getenv("MY_NAMESPACE") or "thoth-test-core"
```